docs/Populate.md

```diff
@@ -0,0 +1,118 @@
+# Populate Exchange
+
+`populate` is an exchange for auto-populating fields in your mutations.
+
+## How to use
+
+As with any exchange, add this to your client (before your cache).
```

Comment:
```suggestion
As with any exchange, add this to your client (before your cache). This ensures it will add the attributes to your mutation selection so it can be properly used in the cache to update fields affected by the mutation.
```

Nit: I personally forget this often but maybe we should give a little context so people can reason easier when it comes to non-default exchanges (they have written themselves and stuff)

Reply:
How about something like this?

```suggestion
As with any exchange, add this to your client.

> Note: Populate needs to be declared before the cache in order to ensure the `populate` directive is replaced with the respective attributes.
```

Reply:
I'd make sure that you say `@populate` directive (with the `@` symbol) and say "respective selection sets" instead of "attributes" to use proper GraphQL terminology, otherwise lgtm :+1: